Stack/Opc.Ua.Core/Opc.Ua.Core.csproj

```diff
@@ -92,6 +92,7 @@
   <PropertyGroup>
     <ZipTmp>$(BaseIntermediateOutputPath)/zipnodeset2</ZipTmp>
     <ZipNodeSet2XML>Schema/Opc.Ua.NodeSet2.xml</ZipNodeSet2XML>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
```

Comment:
in the meantime I heard from various sources that this may be a concern. We will need to do some digging if the unsafe code needs to be in a seperate assembly.